pvmismatch/pvmismatch_lib/pvmodule.py

```diff
@@ -390,8 +429,9 @@ def calcMod(self):
         """
         # iterate over substrings
         # TODO: benchmark speed difference append() vs preallocate space
+        bypass_entire_module = False
```

Comment:
This local variable is never used later. You are already using `module_bypass` for the case of one bypass diode for entire module.